ddpui/core/alerts/alert_service.py

```
@@ -0,0 +1,461 @@
+        count_sql = compile_query(count_qb, wclient)
+
+        # Paginated results
+        paginated_qb = build_paginated_query_builder(base_qb, data.page, data.page_size)
+        paginated_sql = compile_query(paginated_qb, wclient)
+
+        try:
+            count_result = wclient.execute(count_sql)
+            total_rows = count_result[0]["cnt"] if count_result else 0
+
+            results = wclient.execute(paginated_sql) if total_rows > 0 else []
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)